broken_link_checker/checker.py

```diff
@@ -154,14 +145,14 @@ def update_list(self, response: urllib3.response.HTTPResponse) -> None:
         if self.REGEX_CONTENT_TYPE.match(response.headers['Content-Type']):
             self.logging.debug('Getting of the webpage...')
             # we read max 2**20 bytes by precaution
-            data = response.read(1048576)
+            data = response.raw.read(1048576)
             self.logging.debug('Decoding of data...')
             data = data.decode()
 
             # We verify if we are not already got this content in the previous request
             if difflib.SequenceMatcher(None, data, self.prev_data).ratio() > 0.9:
```

Comment:
Salut. Je ne suis pas très en confiance quand je vois ceci.

En fait parcourir toutes les urls d'un site est une sorte de parcours de graphe.
Appelons worked_urls la liste des urls que tu as parcouru.
Chaque fois tu finis de check (self.check()) une url tu dois l'ajouter dans  worked_urls
Chaque fois que tu veux check une url tu dois d'abord verifier qu'elle n'est pas dans la liste worked_urls.

De cette façon tu es sur de passer par une url chaque fois et tu n'auras pas besoin de comparer les contenus de 2 requetes consécutives.

Reply:
Thank you for your review. I think, I told you couple of times already that it is mandatory to communicate here in english. So I am kindly asking you to translate your message in english as we would like to keep that consistency everywhere, in the organisation's pull requests and issues and  especially as I know you totally capable of writing in English. We may consider deleting this message from the PR in few days if the translation does no happen. And you won't be welcomed to comment in the org if you persist writing here in french.